andela/troupon
# [Feature #126489763] Generate downloadable receipts for virtual deals

#### What does this PR do?

This PR allows the download and printing of tickets for virtual deals such as concert tickets and the likes.

#### Description of Task to be completed?

After successful payment, the application would check for virtual deals and present them to the user as a list. These links will point to a view that would generate an ID unique to the current transaction. This ID will then be embedded in a QRcode. 

The details of the current virtual transaction being processed along with an image of the QR code will be rendered on an HTML template. This template will be converted to a PDF document and returned to the user.

#### How should this be manually tested?
- Purchase one or more virtual deals.
- Proceed to checkout.
- Pay for the transaction.
- On "payment successful" page, click on the link pointing to the virtual deals.
- Confirm download.
- Open the downloaded file and confirm the receipt has accurate information.

#### What are the relevant pivotal tracker stories?

#126489763

#### Screenshots:
![image](https://cloud.githubusercontent.com/assets/7263503/17484363/582edb68-5d81-11e6-9586-407af2388f3b.png)

![image](https://cloud.githubusercontent.com/assets/7263503/17484350/5030572a-5d81-11e6-9f73-97ac5639ed6d.png)
